src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -74,7 +71,34 @@
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Skip_analyzers_for_implicitly_triggered_builds}" />
                 </StackPanel>
             </GroupBox>
-            
+
+            <GroupBox x:Uid="SourceGeneratorsGroupBox"
+                      Header="{x:Static local:AdvancedOptionPageStrings.Option_Source_Generators}">
+                <StackPanel>
+  
+                    <StackPanel Margin="0, -5, 0, 5">
+                        <Label x:Name="Run_source_generators_label" Content="{x:Static local:AdvancedOptionPageStrings.Option_Source_generator_execution_requires_restart}"/>
```

Comment:
❓ Are we sure this requires a restart? I would expect it to be rather straightforward to handle this immediately when the value changes.

Reply:
Yes.  Currently this changes a WorkspaceConfigurationOption value, which requires restarts currently, as the value is cached.  If someone wants to fixup that system to apply immediately, i'd have no issue with that.